visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -54,9 +54,11 @@
 #include <pcl/visualization/interactor_style.h>
 
 // VTK includes
+class vtkSmartPointerBase;
+template <class T>
+class vtkSmartPointer;
 class vtkPolyData;
 class vtkTextActor;
-class vtkRenderWindow;
```

Comment:
This declaration is used in this header. I'm surprised the CI is not complaining 😕